sigs.yaml

```diff
@@ -3583,6 +3583,41 @@ workinggroups:
     liaison:
       github: aojea
       name: Antonio Ojea
+- dir: wg-checkpoint-restore
+  name: Checkpoint Restore
+  mission_statement: >
+    This working group aims to provide a central location for the community to discuss
+    the integration of Checkpoint/Restore functionality into Kubernetes.
+
+  charter_link: charter.md
+  stakeholder_sigs:
```

Comment:
sig auth may have a big say in security of this whole restoration pipeline

Reply:
nod.. definitely needs an extra level of security due to customer data being serialized and available in the checkpoint, esp if not encrypted

Reply:
(to close the loop on the sig-auth bit, there was another round of feedback in the [2025-10-22 sig-auth meeting](https://docs.google.com/document/d/1woLGRoONE3EBVx-wTb4pvp4CI7tmLZ6lS26VTbosLKM/edit?tab=t.0#bookmark=id.qrccz54wxt2n))

glad to see folks signed up to work in this area, there's a lot that's useful here, but also a lot of novel interactions to pin down before exposing the capabilities to anyone other than node root users via APIs